app/serializers/book_ranking_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class BookRankingSerializer < Panko::Serializer
```

Comment:
you should inherit of BookSerializer and only define here the additional attributes that you need